go-controller/hybrid-overlay/pkg/controller/master_test.go

```diff
@@ -23,15 +24,26 @@ import (
 
 const hoNodeCliArg string = "-no-hostsubnet-nodes=" + v1.LabelOSStable + "=windows"
 
-func addGetPortAddressesCmds(fexec *ovntest.FakeExec, nodeName, hybMAC, hybIP string) {
+func mockPortAddressesCmds(nodeName, hybMAC, hybIP string) {
```

Comment:
The fact that this involves "commands" is now purely an implementation detail. It should be called something like "`setInitialPortAddresses`" or something

(EDIT: `populatePodDynamicAddresses` in one of the other tests is a good example of a better name)

Reply:
agreed. will change the names.